### PR TITLE
Some more API updates

### PR DESCRIPTION
**Pull request by nc6**

### CODE REVIEW

File: shelley/chain-and-ledger/executable-spec/src/Cardano/Ledger/Shelley/API/Mempool.hs [hunk 1]
```diff
@@ -12,6 +13,7 @@ module Cardano.Ledger.Shelley.API.Mempool
     mkMempoolState,
     ApplyTxError (..),
     applyTxs,
+    overShelleyState,
```
Comment (mgudemann) [inline shelley/chain-and-ledger/executable-spec/src/Cardano/Ledger/Shelley/API/Mempool.hs]: trailing comma?

File: shelley/chain-and-ledger/executable-spec/src/Keys.hs [hunk 2]
```diff
@@ -47,7 +47,7 @@ module Keys
   , SKeyES(..)
   , VKeyES(..)
   , KeyHashES(..)
-  , KESig
+  , KESig(..)
```
Comment (mgudemann) [inline shelley/chain-and-ledger/executable-spec/src/Keys.hs]: not sure where this is used

Comment (nc6) [reply shelley/chain-and-ledger/executable-spec/src/Keys.hs]: In ouroboros-consensus

Comment (mgudemann) [reply shelley/chain-and-ledger/executable-spec/src/Keys.hs]: ok, thanks!

**Pull request merged**